core/templates/dev/head/css/oppia.css

```diff
@@ -858,6 +858,13 @@ textarea {
     margin-top: 0;
     margin-left: 5px;
   }
+  ul.nav.navbar-nav.oppia-navbar-nav.navbar-right {
+    display: flex;
+    float: right;
+    margin: 0;
+  }
+}
+@media(max-width: 651px) {
```

Comment:
@Oishikatta 
Oh, strange. This was the error with my first commit. With the second commit, it was resolved. At my end, everything seems (with the latest committed branch).
Could you please check again ?
**Latest commit:**

![screenshot from 2017-04-25 19-31-59](https://cloud.githubusercontent.com/assets/24438869/25389428/3ccc4ccc-29ee-11e7-86c8-63927b10cdcc.png)